### PR TITLE
only add temporal

### PR DESCRIPTION
**Pull request by jrhizor**
Extract only the parts of https://github.com/airbytehq/airbyte/pull/2130/files needed to run temporal.

todo:
- [x] clean up the docker compose file
- [x] improve config defaults
- [x] use same db container
- ~~add a getter for the client that waits until the temporal service is available~~ couldn't replicate

### CODE REVIEW

**Comment by jrhizor** [comment]
We don't need to create the admin tools if we run `docker run --rm --network=host temporalio/tctl:1.7.0 COMMAND` separately. We should document this in the temporal maintenance section.

We can also run the web UI on demand with `docker run --rm -p 8088:8088 temporalio/web:1.7.0`

**Review by michel-tricot — APPROVED** [review]
LGTM

File: airbyte-scheduler/build.gradle [hunk 1]
```diff
@@ -3,6 +3,8 @@ plugins {
 }
 
 dependencies {
+    implementation 'io.temporal:temporal-sdk:1.0.4'
```
Comment (davinchia) [inline airbyte-scheduler/build.gradle:6]: question for my own understanding: we place external dependencies above internal ones?

Comment (cgardens) [reply airbyte-scheduler/build.gradle:6]: i think we actually usually do internal and then external.

(within each group we alphabetize)
```
main internal

main external

test internal

test external

integrationTest internal

integrationTest external
```

**Review by cgardens — APPROVED** [review]

**Pull request merged**